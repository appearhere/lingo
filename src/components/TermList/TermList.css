.container {
  display: grid;
  grid-gap: var(--size-regular);
  min-height: 0;
  min-width: 0;
  margin-bottom: var(--size-regular);
}

.h2 {
  display: none
}

.item {
  overflow: hidden;
  min-width: 0;
  min-height: 0;
}

@media(min-width: 36.25rem) {
  .container {
    grid-template-columns: repeat(2, 1fr);
    margin-bottom: var(--size-lg-iii);
  }

  .h2 {
    margin: 0;
    display: block;
    font-size: var(--fontsize-large-i);
    font-weight: var(--fontweight-demi);
  }

  .h2:after {
    margin-top: var(--size-regular);
    margin-bottom: var(--size-regular);
    background-color: var(--color-black);
    height: 1px;
    bottom: 0;
    content: '';
    display: block;
    left: 0;
    vertical-align: middle;
    width: 100%;
  }
}
