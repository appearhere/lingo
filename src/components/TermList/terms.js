export default [
  {
    name: 'TTV',
    department: 'Concierge',
    definition: 'Total transaction value',
  },
  {
    name: 'Available days',
    department: 'Supply',
    definition: 'The number of bookable days for a space',
  },
  {
    name: 'PPC',
    department: 'Marketing',
    definition: 'Price per click',
  },
];
