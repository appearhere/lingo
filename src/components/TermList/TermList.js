import React from 'react';

import terms from './terms';

const TermList = () => (
  <div>
    { terms.map((term) => <div key={ term.name }>{ term.name }</div>) }
  </div>
);

export default TermList;
