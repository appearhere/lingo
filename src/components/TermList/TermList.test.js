import React from 'react';
import renderer from 'react-test-renderer';
import TermList from './TermList';

describe('TermList component', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<TermList />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
