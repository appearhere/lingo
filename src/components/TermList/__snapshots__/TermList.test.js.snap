// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TermList component renders correctly 1`] = `
<section>
  <div
    className={undefined}
  />
</section>
`;
